articles/understanding-differential-privacy/index.md

```diff
@@ -0,0 +1,297 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/understanding-differential-privacy/
+title: Understanding Differential Privacy
+description: Introduction to differential privacy and methods used to preserve privacy in Databases
+author: srishilesh-p-s
+date: 2020-10-06T00:00:00-12:00
+topics: [Security]
+excerpt_separator: <!--more-->
+images:
+  - url: /engineering-education/understanding-differential-privacy/hero.jpg
+    alt: Understanding Differential Privacy
+---
+
+In this article, we will get an overview of resolving the privacy issue using Differential privacy. You will understand the basics of how privacy is preserved in Databases used for Machine learning and Deep learning. Though understanding differential privacy requires a Mathematics background, this article covers a very basic overview of the concepts. You will also be coding a sample database and check if it is differentially private or not.
+<!--more-->
+
+### Table of contents
+
+- [Introduction](#introduction)
+- [Differential Privacy](#differential-privacy)
+- [Differencing attacks](#differencing-attacks)
+- [Implementing differential privacy](#implementing-differential-privacy)
+- [Types of Differential privacy](#types-of-differential-privacy)
+- [Conclusion](#conclusion)
+- [Further reading](#further-reading)
+
+### Introduction
+
+In this digital era, most of the companies are data-driven. Our data is harvested to keep growing their economic incentives, in exchange for services provided to us. Research institutions often use and share data containing confidential information about individuals. Improper disclosure of such data can have adverse consequences for a data subject’s private information, or even lead to civil liability or bodily harm.
```

Comment:
or bodily harm.

Can you elaborate on this

Reply:
Sure, I have made the statement clear, and have added hyperlinks redirecting to the sources